### ISSUE TITLE
文本识别训练时提示：The shape of model params head.fc.weight [512, 8423] not matched with loaded params head.fc.weight [512, 6625]

### ISSUE THREAD

**Issue by skw1975** [PaddlePaddle/PaddleOCR#4357]
配置文件使用configs/rec/ch_ppocr_v2.0/rec_chinese_common_train__v2.0.yml，只改了文件路径，预训练模型使用的是https://paddleocr.bj.bcebos.com/dygraph_v2.0/ch/ch_ppocr_server_v2.0_rec_pre.tar，为什么会提示如下信息，有没有什么影响 ？
[2021/10/15 14:30:50] root INFO: The shape of model params head.fc.weight [512, 8423] not matched with loaded params head.fc.weight [512, 6625] !
[2021/10/15 14:30:50] root INFO: The shape of model params head.fc.bias [8423] not matched with loaded params head.fc.bias [6625] !
[2021/10/15 14:30:50] root INFO: loaded pretrained_model successful from ./pretrain_models/ch_ppocr_server_v2.0_rec_pre/best_accuracy.pdparams
[2021/10/15 14:30:50] root INFO: train dataloader has 920 iters
[2021/10/15 14:30:50] root INFO: valid dataloader has 97 iters
[2021/10/15 14:30:50] root INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
[2021/10/15 14:30:50] root INFO: Initialize indexs of datasets:['/home/aistudio/train_data/train_label.txt']
[2021/10/15 14:31:52] root INFO: epoch: [1/500], iter: 10, lr: 0.001000, loss: 104.030624, acc: 0.000000, norm_edit_dis: 0.0.........................

**Comment by skw1975** [comment]
找到原因了，字典使用的不对，难道不能中英文混合训练吗？

**Comment by Gmgge** [comment]
中英文是可以混合训练的，你可以跑一下你的标签，用set去重，看下最后字典和ppocr_keys_v1.txt这个字典对比，应该就能发现不同了，然后自己创建一个新的字典。
